Makefile

```diff
@@ -41,10 +42,12 @@ AUTHOR ?= $(shell git config user.email || echo $$USER)
 ENABLE_METRICS ?= true
 BUILD_FLAGS ?= $(shell echo "-ldflags='\
 	-X github.com/status-im/status-go/params.Version=$(RELEASE_TAG:v%=%) \
+	-X github.com/status-im/status-go/params.IPFS_GATEWAY_URL=$(IPFS_GATEWAY_URL) \
```

Comment:
Purely an aesthetic nitpick, but putting it between `Version` and `GitCommit` is kinda weird.

Reply:
I will take the liberty of fixing this before merging.